tools/debug/debugsymbolviewer.py

```diff
@@ -177,8 +177,8 @@ def print_symbol(stack_addr, search_addr, is_app_symbol):
 		if debug:
 			print('Symbol not found for address: {0} in map file'.format(hex(search_addr)))
 
-# Function to search the search_addr for corrupted heap node present in the Address to Symbol mapping table and print its corresponding symbol
-def print_symbol_heap_corruption(search_addr, is_app_symbol):
+# Function to search the search_addr present in the Address to Symbol mapping table and print its corresponding symbol only
+def print_symbol_only(search_addr, is_app_symbol):
```

Comment:
Do you need to keep two separated function for printf difference?
Is it possible to merge them with 1 more argument for the difference?

Reply:
Yes, it is better to have two since both have varied:
1) Arguments
2) Format of printing

Below are their examples:
```
Current running work function		File_name
0xe041e81				/root/tizenrt/apps/da_module/da_common_event_pubsub/src/da_common_event_pubsub.c:162
```
```
Stack_address	 Symbol_address	 Symbol location  Symbol_name		File_name
0x10064798	 0x10013014	 kernel  	  sem_unblock_task	/root/tizenrt/os/kernel/semaphore/sem_post.c:96
0x100647c0	 0xe005d60	 kernel  	  mm_mallinfo	/root/tizenrt/os/include/tinyara/mm/mm.h:643
0x100647f0	 0xe00591c	 kernel  	  mm_manage_alloc_fail	/root/tizenrt/os/include/tinyara/mm/mm.h:726
0x10064828	 0xe0053d8	 kernel  	  malloc	/root/tizenrt/os/include/stdlib.h:453
0x10064830	 0xe1531f4	 kernel  	  __func__.8648
```

Reply:
I think we can still make common function? stack_addr = 0x00000000 set for print_symbol so we can pass some negative value and check to separate it.

Reply:
modified and merged into one API